doc/source/serve/http-guide.md

```diff
@@ -1,12 +1,37 @@
-# Calling Deployments via HTTP
+# HTTP with Serve
 
 
+
+This section should help you:
+
+- Understand how to configure HTTP server location
+- Understand how to send HTTP request to serve deployment
+- Understand how Ray Serve integrate with FastAPI
+- Understand how to use customized HTTP Adapter
+
+
+## Configuring HTTP Server Locations
+
+By default, Ray Serve starts a single HTTP server on the head node of the Ray cluster.
+You can configure this behavior using the `http_options={"location": ...}` flag
+in {mod}`serve.start <ray.serve.start>`:
+
+- "HeadOnly": start one HTTP server on the head node. Serve
+  assumes the head node is the node you executed serve.start
+  on. This is the default.
+- "EveryNode": start one HTTP server per node.
+- "NoServer" or `None`: disable HTTP server.
+
```

Comment:
Make these bold **HeadOnly**, **EveryNode**, **NoServer**, and remove the quotes.